tests/unit/parser/test_unit_tests.py

```diff
@@ -5,9 +5,10 @@
 from dbt.contracts.graph.unparsed import UnitTestOutputFixture
 from dbt.parser import SchemaParser
 from dbt.parser.unit_tests import UnitTestParser
-from tests.unit.test_parser import SchemaParserTest, assertEqualNodes
 from tests.unit.utils import MockNode
 
+from .test_parser import SchemaParserTest, assertEqualNodes
```

Comment:
NIT: Instead of a relative import, we could do `from tests.unit.parser.test_parser import SchemaParserTest, assertEqualNodes`. The only real benefit though is it'll be grouped better with the rest of the imports, so not important at all.

Reply:
Updated!